src/Altinn.Platform/Altinn.Platform.Storage/IntegrationTest/appsettings.json

```diff
@@ -0,0 +1,31 @@
+{
+  "AzureCosmosSettings": {
+    "EndpointUri": "https://localhost:8081",
+    "PrimaryKey": "C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==",
+    "Database": "ServiceEngine",
```

Comment:
Database has changed name to Storage in settings.

Reply:
I deleted the file entirely. Can be recreated when we write tests that needs settings different from the actual app.